tests/test_moe_fp8_dnn.py

```
@@ -0,0 +1,124 @@
+Uses pre-converted DeepSeek-V2-Lite FP8 model (.asgraph + .asparam).
+The new kernel is activated when use_dnn_=true (SM9.0) and is_fp8_=true.
+
+Usage:
+  LD_PRELOAD=/usr/local/cuda-12.8/targets/x86_64-linux/lib/libcudart.so.12 \
+  CUDA_VISIBLE_DEVICES=0 python tests/test_moe_fp8_dnn.py \
+    --model_dir /scratch/jzhang/dashinfer_dsv2_lite_fp8 \
+    --model_name DeepSeek-V2-Lite
+"""
+
+import argparse
+import os
```

Flake8 (7.3.0)

[error] 'os' imported but unused

(F401)